### ISSUE TITLE
should One-shots have no series or a series of the same name?

### ISSUE THREAD

**Issue by gotson** [anansi-project/rfcs#32]
At the moment in ComicVine, one shots are contained in a series of the same name ([example](https://comicvine.gamespot.com/seconds/4050-75753/)).

Does it bring any value to have a series with a single book?

**Comment by bpepple** [comment]
I tend to think so. Would you associate publisher, volume, and other data at an issue level? To me it seems to make more sense to do it at a Series level (though the nomenclature of using 'Series' might not be appropriate).

Over at Metron, I use a series type to differentiate between graphic novel, one-shot, ongoing, etc. ([example](https://metron.cloud/series/i-am-not-starfire-2021/))

**Comment by gotson** [comment]
Take 2000 AD as an example, the publisher changed during the publication, but it's still a single series. In CV they only show the last publisher, and add a comment in the description to explain what the other publishers were for which number.

Volume is tied to Series anyway.

> Over at Metron, I use a series type to differentiate between graphic novel, one-shot, ongoing, etc. ([example](https://metron.cloud/series/i-am-not-starfire-2021/))

There's another discussion on #31 about where to put different type of books (TPB, floppy, omnibus…) for the same series. You don't seem to have TPB in Metron yet, so maybe you haven't decided on that point yet?

**Comment by bpepple** [comment]
> There's another discussion on #31 about where to put different type of books (TPB, floppy, omnibus…) for the same series. You don't seem to have TPB in Metron yet, so maybe you haven't decided on that point yet?

Yeah, I've held off on adding TPB's yet since I want to give more thought about how to implement it (like linking issues to trades and so forth).

**Comment by jakem742** [comment]
If the Book -> Edition -> Series model was used, it could make sense to think of one-shots as a unique edition within a series? Simply set the Edition's mediaType="One-Shot" and series="parentSeries" and in the Book object reference the Series event number. In this case, the parent series would have a numbered list of events (including one-shots?). Refer [here](https://github.com/anansi-project/rfcs/issues/21#issuecomment-1113911945) for my comments on numbered events.

**Comment by GlassedSilver** [comment]
Has there ever been a case of a manga getting announced and maybe even released as one-shot and turned into a series or at least getting a second book later on? Maybe the data model should be able to gracefully handle such oddballs as well without necessarily changing the first book‘s metadata.
